node/src/components/contract_runtime.rs

```diff
@@ -134,6 +151,33 @@ impl ExecutionPreState {
 
 type ExecQueue = Arc<Mutex<BTreeMap<u64, (FinalizedBlock, Vec<Deploy>, Vec<Deploy>)>>>;
 
+#[derive(Debug, From, Serialize)]
+pub(crate) enum Event {
+    // TODO: Consider boxing the internals of `ContractRuntimeRequest` to make the variant smaller
```

Comment:
The `TODO` can be removed here, if this code persists as-is.

Reply:
No boxing is needed with the introduction of https://github.com/casper-network/casper-node/commit/161222fffeefa683ccbfd5e57e554d59cefdb734